source/basic.tex

```diff
@@ -300,38 +300,38 @@
 an unevaluated operand\iref{expr.prop},
 a subexpression thereof, or
 a conversion in an initialization or conversion sequence in such a context.
-The set of \defn{potential results} of an expression \tcode{e} is
+The set of \defn{potential results} of an expression $E$ is
 defined as follows:
 \begin{itemize}
-\item If \tcode{e} is an
+\item If $E$ is an
 \grammarterm{id-expression}\iref{expr.prim.id}, the set
-contains only \tcode{e}.
-\item If \tcode{e} is a subscripting operation\iref{expr.sub} with
+contains only $E$.
+\item If $E$ is a subscripting operation\iref{expr.sub} with
 an array operand, the set contains the potential results of that operand.
-\item If \tcode{e} is a class member access
+\item If $E$ is a class member access
 expression\iref{expr.ref} of the form
 \tcode{e1 . \opt{template} e2}
```

Comment:
Can we also convert the `e1` and `e2` here and below?

Reply:
Fixed.